entrypoint.sh

```diff
@@ -40,4 +40,14 @@ fi
 
 echo
 echo "Running command '${oryxCommand}'"
+
+url="https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}/jobs"
+
+#Format: "2020-02-15T02:51:50.000Z"
+startTime=$(curl -X GET "${url}" | sed 's/,/\n/g' | grep "started_at" | awk '{print $2}' | sed -n '3p')
```

Comment:
@qianz2 since we are listing all of the jobs within a given run, we can't make any assumptions about the ordering of the steps within each job -- ideally we would identify the step that builds our action (which could be more than one if it's being used in multiple jobs for a given run), parse the start and end times from that specific step, and then set those values to the environment variables. The name of the step building our action should be "Build microsoft/appservice-build@v1", but we can look for some substring of that in case the user is consuming it from a different branch, or if GitHub changes the format of this name.

Reply:
@cormacpayne I realized that a given run usually has "set up a job" and "Build microsoft/appservice-build@v1"(or a different branch name) these 2 fixed steps above the steps that we defined in action.yml and "Post cloning repo" & "Complete job" after the customized steps. I'm wondering if this pattern applies to all runs, could we extract the 3rd occurrence(the first occurrence is the time of entire run) of "started_at" and "completed_at" like I did above? I'm trying to find some counterexamples if it's not applied to all runs.

Reply:
@qianz2 I don't think we can make any assumptions about the ordering of the steps -- it may be the case the a user adds multiple actions into their workflow which GitHub will need to clone and build, but we don't know what order that they will do that in. It's safer if we can parse the steps to determine which is building our action and set the start and end time reported in that step.